src/components/date_filter/DateFilter.tsx

```diff
@@ -0,0 +1,36 @@
+import CalendarIcon from "../../assets/calendar-icon.png";
+import { DateFilterContainer, DatePickerWrapper } from "./DateFilter.styles";
+import InputDate from "../date/inputDate";
+
+interface DateFilterProps {
+  value: Date | null | undefined;
+  onChange: (date: Date | null | undefined) => void;
+}
+
+const DateFilter: React.FC<DateFilterProps> = ({ value, onChange }) => {
+  return (
+    <DateFilterContainer>
+      <DatePickerWrapper $hasValue={!!value}>
+        <InputDate
+          onChange={(date) => {
+            onChange(date instanceof Date ? date : null);
+          }}
+          value={value}
+          format="dd/MM/yyyy"
+          locale="es-ES"
+          clearIcon={null}
+          calendarPosition="filter"
+          calendarIcon={
+            <img
+              src={CalendarIcon}
+              alt="Calendario"
+              style={{ width: 20, height: 20 }}
```

Comment:
```suggestion
             style={{ width: 24, height: 24 }}
```

Agrandaría un poco el ícono del calendario